test/helpers/cons.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var (
 	// HelperTimeout is a predefined timeout value for commands.
-	HelperTimeout = 5 * time.Minute
+	HelperTimeout = 8 * time.Minute
```

Comment:
:(

Reply:
Hopefully the CI boxes don't need to rely in this, this was needed for my local ginko runs.

Reply:
This PR should not be merged with this change. If it takes longer than 5 minutes for pods to initialize, then the cause should be determined.

Reply:
I'll try leaving it out. As said it was needed for local ginkgo run, and my internet connection may have been slower than normal at the time, etc.